params/robot_localization_config.yaml

```diff
@@ -150,6 +156,15 @@ twist0_config: [false, false, false,
 twist0_queue_size: 3
```

Comment:
Other camera estimates have much larger queues making this inconsistent. Is there a reason for that?

Reply:
Yes, because this is binary.  It's either 0 or not, we don't really care about buffering a ton of 0's

Reply:
I'm talking about if the filter needs to recalculate old points.

Reply:
This doesn't affect that